consensus/parlia/parlia.go

```diff
@@ -1512,6 +1555,20 @@ func (p *Parlia) Close() error {
 	return nil
 }
 
+// GetConsensusData load the header's last snapshot for history segment
+func (p *Parlia) GetConsensusData(chain consensus.ChainHeaderReader, header *types.Header) ([]byte, error) {
+	number := header.Number.Uint64()
+	snap, err := p.snapshot(chain, number-1, header.ParentHash, nil)
```

Comment:
why exclude current header？
from interface name, It's try to get GetConsensusData from chain with header as latest block